assets/js/bundle.js

```diff
@@ -9597,10 +9597,10 @@
 			fadeTo: function fadeTo(speed, to, easing, callback) {
 
 				// Show any hidden elements after setting opacity to 0
-				return this.filter(isHidden).css("opacity", 0).show()
+				return this.filter(isHidden).css("opacity", 0).show
 
 				// Animate to the value specified
-				.end().animate({ opacity: to }, speed, easing, callback);
+				().end().animate({ opacity: to }, speed, easing, callback);
```

Comment:
Reason for this?

Reply:
No Idea, I think when I ran `bundle install` this happened or maybe my text-editor linter.

I'll change it back

Reply:
That's fine then, so as it's functional.